### ISSUE TITLE
inform screen reader of notebook inline suggestion

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#176288]
We should play an audio cue and will also want the screen reader to read the content immediately like we will do for the editor tracked by https://github.com/microsoft/vscode/issues/167348

**Comment by meganrogge** [comment]
@rebornix @amunger what are your thoughts here/ has there been any headway made on this?

**Comment by amunger** [comment]
@meganrogge - I assume we should follow what we did for https://github.com/microsoft/vscode/issues/167348, but it's hard to follow what the result was there. It looks like the last PR that was referenced was reverted, and I don't see the applicable code in the `ghostTextWidget`. Was there a final decision on behavior somewhere or just a TPI?  And is there another PR that I could use for reference?

**Comment by meganrogge** [comment]
Hi Aaron, we control the one in the editor with a setting. here's the implementation of that 

https://github.com/microsoft/vscode/blob/dcafd8876c9365ca3fc6fe0f449aaa9a16570c15/src/vs/editor/contrib/inlineCompletions/browser/inlineCompletionsController.ts#L236-L238

**Comment by amunger** [comment]
the same code is hit for a notebook cell, the notification sound is played for me, and it looks like the aria alert is being added.
![image](https://github.com/microsoft/vscode/assets/2019016/c9a42e6f-6735-4d26-99a7-aa6452c2ef9f)

I don't ever hear any alert text on either of my machines and haven't been able to figure out why so I can't check, but is it not reading the ghost text for notebook cells?
Is there something else that's missing?

**Comment by meganrogge** [comment]
Can you use the NVDA speech viewer to verify the alert is happening? @amunger

**Comment by meganrogge** [comment]
what is the value of `lineText` when that gets hit?

**Comment by amunger** [comment]
`lineText` is whatever else is on the line.
On my laptop the text editor ghost text is not read either.

I tried again on my desktop pc, and now the ghost text is read for both notebook cells and text editors. Are you seeing something different?

**Comment by meganrogge** [comment]
Sounds like an issue with the screen reader/ setup you have. I would suggest using the speech to text viewers rather than the audio version as then you won't wonder if it's an issue with your setup or the code.
